src/platform/forms-system/src/js/review/ReviewChapters.jsx

```diff
@@ -11,44 +10,16 @@ import {
   getActiveChapters,
   getPageKeys,
 } from '../helpers';
-import { getReviewPageOpenChapters, getViewedPages } from '../state/selectors';
-import {
-  closeReviewChapter,
-  openReviewChapter,
-  setData,
-  setEditMode,
-  setViewedPages,
-  uploadFile,
-} from '../actions';
+import { getViewedPages } from '../state/selectors';
+import { setData, setEditMode, setViewedPages, uploadFile } from '../actions';
 
-const scroller = Scroll.scroller;
 class ReviewChapters extends React.Component {
   componentDidMount() {
     const { formData, pageList } = this.props;
     const viewedPages = new Set(getPageKeys(pageList, formData));
     this.props.setViewedPages(viewedPages);
   }
 
-  handleToggleChapter({ name, open, pageKeys }) {
-    if (open) {
-      this.props.closeReviewChapter(name, pageKeys);
-    } else {
-      this.props.openReviewChapter(name);
-      this.scrollToChapter(name);
-    }
-  }
-
-  scrollToChapter = chapterKey => {
-    scroller.scrollTo(
-      `chapter${chapterKey}ScrollElement`,
-      window.Forms?.scroll || {
-        duration: 500,
-        delay: 2,
-        smooth: true,
-      },
-    );
-  };
```

Comment:
Moved all of this into the chapter component. It didn't need to be in the parent.